index.html

```diff
@@ -4,11 +4,11 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Document </title>
     <link rel="stylesheet" href="style.css"> 
 </head>
 <body>
-    <h1>Hello  Microverse </h1>
+    <h1>Hello Microverse </h1>
 
     
```

Comment:
## 1. 
![PR-TITLE](https://user-images.githubusercontent.com/93039359/184925842-aac97221-6f54-4a55-8105-532937ce657a.png)

- [ ] Your PR has a title that is not so descriptive enough. Because **read me updated** is not describing what are you doing in the project. PR title fully describes what is happening in this branch. You could use **Setup Hello Microverse Project** for your PR title. :+1: